src/constraint/spec.ts

```diff
@@ -102,6 +102,21 @@ export const SPEC_CONSTRAINTS: SpecConstraintModel[] = [
     }
   },
   {
+    name: 'aggregateOnly',
+    description: 'Omit raw plots.',
+    properties: [Property.AGGREGATE, Property.AUTOCOUNT],
+    allowEnumSpecForProperties: true,
+    strict: false,
+    satisfy: (specM: SpecQueryModel, schema: Schema, opt: QueryConfig) => {
+      if (opt.aggregateOnly) {
```

Comment:
You don't need to check for this.  Our generation would trigger this constraints only if it is true.